packages/client/src/consts.js

```diff
@@ -44,6 +44,13 @@ export const FORCE_LAYDOWN = 'ForceLaydown'
 export const VISIBILIY_CHANGES = 'VisibilityChanges'
 export const PERCEPTION_OF_CONTACT = 'PerceptionOfContact'
 export const SUBMIT_PLANS = 'SubmitPlans'
+export const STATE_OF_WORLD = 'StateOfWorld'
+
+// time period to wait if server returns an error. One frequent cause of error
+// during development is that the server is stopped.  We're introducing a
+// throttle value to prevent the browser going into a race condition
+// as it sends 1000s of requests to the server
+export const ERROR_THROTTLE = 3000
 
 // Nov 2019. Ian modified the server path to use the
```

Comment:
I just noticed this, I know it's not from this commit, is this a note to highlight a hack? If so then it would be good to use a comment flag. The ones we should be using are as follows:

`//TODO:` Indicates an item which is still to do in the file
`//REFACTOR:` Indicates an area of code which could be written better, this is generally when you yourself have written something quickly and know it can be more efficiently written rather than to comment on someone else's code.
`//FIXME:` Indicates a bug or other problem which needs to be fixed but can't be done now
`//HACK:` Sometimes we just need to get stuff working and we have to do it in a way we're really not happy with. We can highlight them with the HACK flag.
`//NOTE:` This one is used for when you want to explain a decision or a reason for some code but not actually document the code itself.

These will be added to my ways of working guidelines when I start on the rebuild so it's up to you if you want to add that now or not.

Reply:
Sure, added //NOTE: